editor/js/Agent.js

```diff
@@ -192,7 +192,7 @@ class Agent {
 	async init() {
 
 		// Initialize Google AI
-		const ai = new GoogleGenAI( { apiKey: 'GENAI_API_KEY' } );
+		const ai = new GoogleGenAI( { apiKey: 'AIzaSyAHEPxUn9Ow4Lk8oXM4jl3u_stzOIxkkGM' } );
```

Comment:
@mrdoob any problem on exposing this key?

Reply:
I was exposing it intentionally, but seems like githack doesn't like serving stuff with keys...